build.zig

```diff
@@ -1236,6 +1236,17 @@ fn addDeps(
     step.addCSourceFiles(.{ .files = &.{"src/stb/stb.c"} });
     if (step.rootModuleTarget().os.tag == .linux) {
         step.addIncludePath(b.path("src/apprt/gtk"));
+        step.addIncludePath(std.Build.LazyPath { .cwd_relative = "/usr/include/gtk-4.0" });
+        step.addIncludePath(std.Build.LazyPath { .cwd_relative = "/usr/include/glib-2.0" });
+        step.addIncludePath(std.Build.LazyPath { .cwd_relative = "/usr/lib/x86_64-linux-gnu/glib-2.0/include" });
```

Comment:
If we make a PR upstream we should use a format string for the OS triple, as done here: https://github.com/ghostty-org/ghostty/blob/main/build.zig#L1230